builder/googlecompute/config.go

```diff
@@ -657,24 +663,17 @@ func ApplyIAPTunnel(c *communicator.Config, port int) error {
 	}
 }
 
+var labelKeyRegex = regexp.MustCompile(`^\p{Ll}(\p{Ll}|[0-9_-]){0,62}$`)
```

Comment:
If I'm not mistaken, you should be able to slightly simplify the regex: 
```suggestion
var labelKeyRegex = regexp.MustCompile(`^\p{Ll}[\p{Ll}0-9_-]{0,62}$`)
```

Reply:
Yeah that could work if the grammar allows us using character classes in an interval, but that's easily testable, I'll try that locally, and if it works, we can roll with it

Reply:
OK I tested locally, seems to be working alright indeed, thanks for the suggestion, we should be good to go with this change!